launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -399,6 +399,24 @@ QStringList MinecraftInstance::extraArguments()
     if (version->getModLoaders().value() & ResourceAPI::Quilt && settings()->get("DisableQuiltBeacon").toBool()) {
         list.append("-Dloader.disable_beacon=true");
     }
+
+#ifdef Q_OS_LINUX
+    {
+        QString openALPath;
+        QString glfwPath;
+
+        if (settings()->get("UseNativeOpenAL").toBool())
+            openALPath = MangoHud::findLibrary("libopenal.so");
+        if (settings()->get("UseNativeGLFW").toBool())
+            glfwPath = MangoHud::findLibrary("libglfw.so");
+
+        if (!openALPath.isEmpty())
+            list.append("-Dorg.lwjgl.openal.libname=" + openALPath);
+        if (!glfwPath.isEmpty())
+            list.append("-Dorg.lwjgl.glfw.libname=" + glfwPath);
+    }
+#endif
```

Comment:
Would this feature not be potentially useful on other platforms, like Windows and macOS? Or is there some other reason? I know Windows is Not Great in regards to this, is that why?

Reply:
Well. There is no implementation for those other platforms yet. This can obviously be expanded in the future, though the main use-case is Linux.